src/query/tsdb/remote/compressed_codecs.go

```diff
@@ -434,3 +443,11 @@ func DecodeCompressedFetchResponse(
 		nil,
 	), nil
 }
+
+// EncodeToCompressedSearchResult encodes SearchResults to a compressed search result
+func EncodeToCompressedSearchResult(
+	results *storage.SearchResults,
+	iterPools encoding.IteratorPools,
+) (*rpc.SearchResponse, error) {
+	return nil, nil
```

Comment:
same nit: Can we declare `var errNotImplemented = errors.New("not implemented")` and `return errNotImplemented` here until implemented?  Then at least it is transparent about behavior.

Reply:
Similar comment in that this PR should eventually implement this function